### PR TITLE
Also test on clients and send email on a triggered build

### PR DESCRIPTION
**Pull request by yarikoptic**
I've not tried.  Might need a thoroughly careful review ;)

### CODE REVIEW

File: .github/workflows/template/build-{{ostype}}.yaml.j2 [hunk 1]
```diff
@@ -356,7 +356,7 @@ jobs:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
 
       - name: Send e-mail on failed scheduled run
```
Comment (jwodder) [inline .github/workflows/template/build-{{ostype}}.yaml.j2:358]:
```suggestion
      - name: Send e-mail on failed run
```

File: .github/workflows/template/build-{{ostype}}.yaml.j2 [hunk 2]
```diff
@@ -519,7 +519,7 @@ jobs:
...
```
Comment (jwodder) [inline .github/workflows/template/build-{{ostype}}.yaml.j2:521]:
```suggestion
      - name: Send e-mail on failed run
```

**Review by jwodder — CHANGES_REQUESTED** [review]

**Comment by jwodder** [comment]
This change is also needed:

```diff
diff --git a/.github/workflows/tools/daily-status.py b/.github/workflows/tools/daily-status.py
index c21da03d0e..a98d571013 100644
--- a/.github/workflows/tools/daily-status.py
+++ b/.github/workflows/tools/daily-status.py
@@ -166,7 +166,7 @@ def main():
         for wfname in WORKFLOWS:
             wf = wfrepo.get_workflow(wfname)
             for run in wf.get_runs():
-                if run.status != "completed" or run.event != "schedule":
+                if run.status != "completed" or run.event not in ("schedule", "workflow_dispatch"):
                     continue
                 dt = ensure_aware(run.created_at)
                 if dt <= cutoff:
```

**Comment by yarikoptic** [comment]
all good now @jwodder ?

**Pull request merged**